CAT/model/IRT.py

```diff
@@ -94,7 +143,13 @@ def adaptest_load(self, path):
         """
         self.model.load_state_dict(torch.load(path), strict=False)
         self.model.to(self.config['device'])
-
+    def adaptest_load_BOBCAT(self, path, policy):
```

Comment:
add args in `adaptest_load` to control whether to load policy instead of a new function